docs/pages/web/sheetmobile.js

```diff
@@ -327,6 +328,27 @@ When using these render props, just pass the argument \`onDismissStart\` to your
             }
           />
         </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="External handlers"
+          description={`SheetMobile consumes external handlers from GlobalEventsHandlerProvider.
+
+Handlers:
+- onOpen: executed on useEffect
+- onClose: executed on the cleanup function of useEffect
+
+See [GlobalEventsHandlerProvider](/web/utilities/GlobalEventsHandlerProvider) for more information.`}
```

Comment:
This URL isn't correct — component names in URLs are always all lowercase

Reply:
DONE